pal/minigame/wechat_mini_program.ts

```diff
@@ -173,4 +173,23 @@ if (systemInfo.platform === 'windows' && versionCompare(systemInfo.SDKVersion, '
     }
 }
 
+// HACK: adapt gl.texSubImage2D: gl.texSubImage2D do not support 2d canvas in wechat miniprogram
+// @ts-expect-error getApp defined in global
```

Comment:
We forbid to use `@ts-expect-error` now.

Reply:
fixed.